build.sbt

```diff
@@ -21,7 +24,15 @@ val scalacheckOps = Seq(
 )
 
 val commonSettings = Seq(
-  organization := "io.mytc",
+  organization := "com.expload",
+
+  licenses += ("Apache-2.0", url("http://www.opensource.org/licenses/apache2.0.php")),
```

Comment:
For now we don't know our license. Can we publish it without license, or it is requirement of bintray?